### ISSUE TITLE
In Miami-Dade County, the /council-members/ URL should be /commissioners/

### ISSUE THREAD

**Issue by ErnieAtLYD** [datamade/django-councilmatic#71]
However, `/council-members/` is hard coded:
https://github.com/datamade/django-councilmatic/blob/master/councilmatic_core/templates/base.html#L36

**Comment by cathydeng** [comment]
yeah, right now we only have settings to customize the display of vocab (`CITY_VOCAB`) but not to customize the urls. we'd welcome a PR that adds takes in url customizations from `settings_jurisdiction.py` (but also doesn't break existing councilmatic instances, & falls back to the defaults we have now)

**Comment by ErnieAtLYD** [comment]
@cathydeng: Are there existing unit/functional tests people run to verify that old URLs work correctly?

**Comment by cathydeng** [comment]
not that I know of. I think it's okay to go without url tests for councilmatic, since we have very few url patterns. I'd approach this by keeping the default url patterns in `councilmatic_core/urls.py`, but adding some extra url patterns based on if url pattern settings are configured, & then updating any hard-coded urls in `councilmatic_core/templates/` accordingly.

at the same time though, I'd consider this low-priority, if you have other things to work on. In Chicago our council members are called aldermen, but we just figured that displaying custom vocab was good enough, without customizing the urls - it's all too easy to go down the rabbit hole of customizing everything :)
